src/dotnet-build/project.json

```diff
@@ -8,7 +7,7 @@
         "NETStandard.Library": "1.0.0-rc3-23727",
         "Microsoft.DotNet.ProjectModel": "1.0.0-*",
         "Microsoft.DotNet.Compiler.Common": "1.0.0-*",
-        "Microsoft.DotNet.Tools.Compiler": "1.0.0-*",
+        "dotnet-compile": "1.0.0-*",
```

Comment:
Slightly unrelated, but it seems odd that `dotnet-build` needs a dependency on `dotnet-compile`